minio/credentials/assume_role.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+# MinIO Python Library for Amazon S3 Compatible Cloud Storage, (C)
+# 2020 MinIO, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from datetime import datetime
+from urllib.parse import urlencode
```

Comment:
I note that `minio.compat` has a `urlencode`... but it is actually just renaming `urllib.quote`/`urllib.request.qote`.

Reply:
Resolved by https://github.com/minio/minio-py/pull/875